### PR TITLE
Relax faraday dependency version

### PR DESCRIPTION
**Pull request by dalibor**
@bhuga Can you please merge and push 0.1.3 to rubygems? Thanks!

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,21 +1,21 @@
 PATH
   remote: .
   specs:
-    faraday-restrict-ip-addresses (0.0.2)
-      faraday (>= 0.8, < 0.9)
+    faraday-restrict-ip-addresses (0.1.2)
```
Comment (bhuga) [inline Gemfile.lock]: The new version os 0.1.3, but this is 0.1.2. I think you need to run `bundle install` again and push the result.

**Comment by dalibor** [comment]
@bhuga Thank you! Just pushed that change.

**Comment by mshappe** [comment]
Bump. This should get merged some day, please. I'm looking at having to fork this repo to update for my own purposes otherwise (which, y'know, that's fine, but you've GOT a patch here...)

**Comment by zenspider** [comment]
I'm gonna close this as stale as well. I believe this to be solved tho.

**Pull request closed without merging**